gliderpy/plotting.py

```diff
@@ -123,3 +127,67 @@ def plot_cast(
     ax.invert_yaxis()
 
     return fig, ax
+
+
+@register_dataframe_method
+def plot_ts(
+    df: pd.DataFrame,
+    profile_number: int,
+) -> tuple[plt.Figure, plt.Axes]:
+    """Make a TS diagram from a chosen profile number.
```

Comment:
Maybe we should do the TS for the whole track. TS diagrams have many uses and the very first one is tp assess data quality, find oddities, etc. I believe that this function will be used to for quick data inspection like that, not really a final figure for a paper or refined research.

Reply:
@ocefpaf , to create a plot with all the profiles, I would need to interpolate them so that they are in the same pressure intervals, adding NaN where the profiles are shallower than others. Do my thoughts make sense, or is there a simpler way?